### PR TITLE
Samyak1080 issue/238

### PR DESCRIPTION
**Pull request by bpauli**
* forked PR from https://github.com/Adobe-Marketing-Cloud/aem-core-wcm-components/pull/252 and activated Junit test for uuid

### CODE REVIEW

File: content/src/content/jcr_root/apps/core/wcm/components/image/v2/image/_cq_design_dialog/.content.xml [hunk 1]
```diff
@@ -86,6 +86,15 @@
                                         text="Display caption as pop-up"
                                         uncheckedValue="false"
                                         value="{Boolean}true"/>
+                                    <disableUuid
+                                        jcr:primaryType="nt:unstructured"
+                                        sling:resourceType="granite/ui/components/coral/foundation/form/checkbox"
+                                        fieldDescription="Check to disable UUID of the asset."
```
Comment (vladbailescu) [inline content/src/content/jcr_root/apps/core/wcm/components/image/v2/image/_cq_design_dialog/.content.xml]: I would reword this to `.. disable tracking of assset's UUID`

File: content/src/content/jcr_root/apps/core/wcm/components/image/v2/image/_cq_design_dialog/.content.xml [hunk 2]
```diff
@@ -86,6 +86,15 @@
...
+                                        name="./uuidDisabled"
+                                        text="Disable UUID"
```
Comment (vladbailescu) [inline content/src/content/jcr_root/apps/core/wcm/components/image/v2/image/_cq_design_dialog/.content.xml]: I would reword this to `Disable UUID tracking`

**Comment by bpauli** [comment]
fixes #238

File: content/src/content/jcr_root/apps/core/wcm/components/image/v2/image/_cq_design_dialog/.content.xml [hunk 3]
```diff
@@ -86,12 +86,12 @@
                                         text="Display caption as pop-up"
                                         uncheckedValue="false"
                                         value="{Boolean}true"/>
-                                    <disableUuid
+                                    <disableUuidTracking
                                         jcr:primaryType="nt:unstructured"
                                         sling:resourceType="granite/ui/components/coral/foundation/form/checkbox"
-                                        fieldDescription="Check to disable UUID of the asset."
+                                        fieldDescription="Check to disable tracking of assset's UUID."
```
Comment (richardhand) [inline content/src/content/jcr_root/apps/core/wcm/components/image/v2/image/_cq_design_dialog/.content.xml]: @bpauli - we introduced a spelling mistake here from Vlad's comment, too many "s"
